python/lsst/pipe/tasks/colorterms.py

```diff
@@ -238,17 +236,11 @@ def getColorterm(self, filterName, photoCatName, doRaise=True):
                         "No colorterm dict found with photoCatName %r" % photoCatName)
             ctDict = ctDictConfig.data
             if filterName not in ctDict:
-                # Perhaps it's an alias
-                try:
-                    filterName = Filter(Filter(filterName).getId()).getName()
```

Comment:
😵

Reply:
I'm not sure what to make of the face? Are you mad at what I removed, or that it was removed?

For the record, the `pass  # this will be handled shortly` just below this was added 6 years ago, so "shortly" in astronomical time, I guess.

Reply:
😵  = "It makes my head spin that the old code made a `Filter` in order to make a `Filter` in order to standardize the filter name."